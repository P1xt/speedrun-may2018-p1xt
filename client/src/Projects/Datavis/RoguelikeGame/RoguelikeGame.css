.RoguelikeGame {
  text-align: left;
  margin-top: 40px; }
  .RoguelikeGame > h1 {
    text-transform: none; }
