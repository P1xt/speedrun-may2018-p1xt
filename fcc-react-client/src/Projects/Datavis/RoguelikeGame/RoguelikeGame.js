import React from "react";
import { Container, Row, Col } from "reactstrap";

import "./RoguelikeGame.css";

const RoguelikeGame = () => {
  return (
    <div className="RoguelikeGame">
      <Container>
        <Row>
          <Col>
            <h1>Roguelike Game</h1>
          </Col>
        </Row>
      </Container>
    </div>
  );
};

export default RoguelikeGame;
