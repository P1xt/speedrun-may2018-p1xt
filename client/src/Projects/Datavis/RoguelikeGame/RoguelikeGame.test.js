import React from "react";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

import RoguelikeGame from "./RoguelikeGame";

describe("<RoguelikeGame />", () => {
  it("renders without crashing", () => {
    shallow(<RoguelikeGame />);
  });
  it("renders correctly", () => {
    const tree = renderer.create(<RoguelikeGame />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
