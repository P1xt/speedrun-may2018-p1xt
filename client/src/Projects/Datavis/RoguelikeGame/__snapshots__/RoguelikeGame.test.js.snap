// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RoguelikeGame /> renders correctly 1`] = `
<div
  className="RoguelikeGame"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col"
      >
        <h1>
          Roguelike Game
        </h1>
      </div>
    </div>
  </div>
</div>
`;
